Replicator/Replicator.cc

```diff
@@ -1091,6 +1067,9 @@ namespace litecore { namespace repl {
 
     // Handles a "setCheckpoint" request by storing a peer checkpoint.
     void Replicator::handleSetCheckpoint(Retained<MessageIn> request) {
+        // 3.0 client may send this message as the first one.
```

Comment:
Incorrect comment

Reply:
Which comment are you referring to? The one above the function header or below it?

Reply:
`// 3.0 client may send this message as the first one.`

Reply:
Actually, I meant it. Because 3.0 client may send this message as the first message, I have to apply the following line. In fact, I mean to add the following line to any handlers that may handle the first message, except for getCollections, which 3.0 client won't send.

Reply:
If you are just adding this comment to everything, the comment value is less.  Have you actually seen a situation in which setCheckpoint comes first?  I can't imagine that it would.

Reply:
I can be 100% certain. Without the certainly, having this line here won't hurt because it would be a no-op after the first call through any other handler.

Reply:
good point. removing them.